skorch/callbacks/training.py

```diff
@@ -398,7 +398,7 @@ def __init__(
 
     def __getstate__(self):
         # Avoids to save the module_ weights twice when pickling
-        state = super().__getstate__()
+        state = self.__dict__
```

Comment:
Could you please use `self.__dict__.copy()`. I believe otherwise you would delete it from the existing object.

Reply:
Yes, good point !